cli/cmd/devtool/start.go

```diff
@@ -258,6 +251,13 @@ func (s cloud) start(ctx context.Context, ch *cmdutil.Helper, verbose, reset, re
 
 	g, ctx := errgroup.WithContext(ctx)
 
+	stateDir := lookupDotenv("RILL_STATE_DIRECTORY")
```

Comment:
All our other env variables have the format `RILL_<service>_<name>`, so should this perhaps be `RILL_DEVTOOL_STATE_DIRECTORY`? Or we could pass it with a flag to the `rill devtool start` instead?